src/warnet/bitcoin.py

```diff
@@ -36,10 +36,12 @@ def rpc(tank: str, method: str, params: str):
 
 
 def _rpc(tank: str, method: str, params: str):
+    # bitcoin-cli should be able to read bitcoin.conf inside the container
+    # so no extra args like port, chain, username or password are needed
```

Comment:
this is v cool, with having bitcoin.conf drive a lot of config via the configmap.